docs/topics/forms/formsets.txt

```diff
@@ -71,7 +71,7 @@ example:
 
     >>> import datetime
     >>> from django.forms import formset_factory
-    >>> from myapp.forms import ArticleForm
+    >>> from .forms import ArticleForm
```

Comment:
Relative imports only work in files, not in a Python console.

Reply:
I don't think it's clear to everyone that what they're looking at here are examples from the python console.

`>>> ` might be a clue to some of us, but other's may just see the python code.

Maybe it needs to be a little clearer;

> In your terminal, start python and create the following form:
> ```
> $ python
> ...
> >>> from django import forms
> >>> class ArticleForm(forms.Form):
> ...     title = forms.CharField()
> ...     pub_date = forms.DateField()
> ```

Reply:
The tutorial covers this. I think adding such boilerplate throughout the rest of the docs is unnecessary. Also, this is just example code, not meant for readers to execute.

Reply:
Yeah I understand that. Just considering how approachable the docs are.